com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -638,6 +642,18 @@ public void Cleanup()
             m_IsInitialized = false;
         }
 
+        /// <summary>
+        /// Get approximate video memory impact, in bytes, of the system.
+        /// </summary>
+        /// <returns>An approximation of the video memory impact, in bytes, of the system<returns>
+        public int GetVMemCost()
```

Comment:
When I read this function name I instinctively think about virtual memory and not video memory. Should we rename it to be clearer? (Maybe it's just me)

Reply:
Oh I can see how it can be indeed confusing, renaming.

Reply:
👍